cmd/sriov-network-config-daemon/start.go

```diff
@@ -86,13 +89,20 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 		panic(err.Error())
 	}
 
+	closeAllConns, err := updateDialer(config)
```

Comment:
As we have this heartbeat check, the 15 timeout setting at line-86 can be removed.

Reply:
Done